lib/registrant.rb

```diff
@@ -0,0 +1,57 @@
+require 'pry'
+
+class Registrant
+  attr_reader :name, 
+              :age, 
+              :permit, 
+              :license_data
+
+  def initialize(name, age, permit = false)
+		@name = name
+		@age = age
+		@permit = permit
+		@license_data = {
+			written: false, 
+			license: false,
+			renewed: false
+		}
+	end
+
+	def permit?
+		@permit
+	end
+
+	def earn_permit
+		@permit = true
+	end
+
+	def pass_written_test
+		if permit? && @age >= 16
+			@license_data[:written] = true
+		end
+	end
```

Comment:
ooooooohhhhh, I see, you put the conditions in this helper method. I would change the name of these methods to something more descriptive then like `pass_written_test?` or `can_take_written?` Also be sure to include the `?` since these methods return boolean values.